rotkehlchen/api/server.py

```diff
@@ -314,6 +315,7 @@
         'per_timestamp_db_snapshots_resource',
     ),
     ('/notes', UserNotesResource),
+    ('/cache/<string:cache_type>/clear', ClearCacheResource),
```

Comment:
This looks different to the endpoint you described in api.rst. There you wrote:  

```
http:post:: /api/(version)/cache/images/<cache_type>/clear
```

```suggestion
    ('/cache/images/<string:cache_type>/clear', ClearImagesCacheResource),
```

So which one is it? Do you need the images or not? I would guess not, since we probably can keep one such endpoint for all types of caches?